packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -245,15 +231,28 @@ export const DatePicker = ({
   )
 
   const modifiers = {
-    start: replaceInvalidDate(startDate),
-    end: replaceInvalidDate(endDate),
-    ...(today ? { today } : {}),
+    start: replaceInvalidDate(startDate) || false,
+    end: replaceInvalidDate(endDate) || false,
+  }
+  const modifiersClassNames = {
```

Comment:
I wonder if this could be declared as a constant at the top just to remove some noise from this function.